src/main/java/org/team1540/base/motionprofiling/RunMotionProfiles.java

```diff
@@ -84,6 +86,17 @@ private double getVelocitySetpoint(MotionProfilingProperties currentProperty, do
     return thisTrajectory.segments[index].velocity / encoderMultiplier * 0.1;
   }
 
+  /**
+   * Sets the velocity to 0 for all properties and sets isFinished to true.
+   */
+  @Override
+  protected void interrupted() {
```

Comment:
Should this be the `end()` method (`interrupted()` calls `end()` by default)? Just in case someone sets a profile that doesn't stop when it ends.

Reply:
No, a profile that doesn't stop the motors when it ends would be the intended behavior in that case.

Reply:
Fair point.